velox/functions/lib/window/tests/WindowTestBase.cpp

```diff
@@ -48,6 +48,78 @@ WindowTestBase::QueryInfo WindowTestBase::buildWindowQuery(
   return {op, functionSql, querySql};
 }
 
+void WindowTestBase::buildOrderByParameters(
```

Comment:
This function needn't be a class method. You can put in a local namespace.

Reply:
Done.